### PR TITLE
Add development README content

### PR DESCRIPTION
**Pull request by cgewecke**
Basically just copying the README over from `setprotocol-v2` to get started here...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,4 +6,100 @@
...
+```
+npm install @setprotocol/index-coop-contracts
+```
+
+[22]: https://www.npmjs.com/package/hardhat
+[23]: https://www.npmjs.com/package/typechain
+
+## Contributing
+We highly encourage participation from the community to help shape the development of Index-Coop. If you are interested in developing on `index-coop` or have any questions, please ping us on [Telegram](https://t.me/joinchat/Fx8D6wyprLUlM1jMVnaRdg).
+
```
Comment (cgewecke) [inline README.md:59]: Is this the right telegram chat for index-coop?

File: README.md [hunk 2]
```diff
@@ -6,4 +6,100 @@
...
+## Security
+
+### TODO: Independent Audits
+
+### Code Coverage
+
+All smart contracts are tested and have 100% line and branch coverage.
+
+### Vulnerability Disclosure Policy
```
Comment (cgewecke) [inline README.md:68]: Does index-coop have it's own vuln disclosure protocol?

File: README.md (hunk 1 above)
Comment (bweick) [reply README.md:59]: I would link the Discord

File: README.md (hunk 2 above)
Comment (bweick) [reply README.md:68]: Same as Set's

File: README.md [hunk 3]
```diff
@@ -6,4 +6,100 @@
...
+
+```
+npm install @setprotocol/index-coop-contracts
+```
+
+[22]: https://www.npmjs.com/package/hardhat
+[23]: https://www.npmjs.com/package/typechain
+
+## Contributing
+We highly encourage participation from the community to help shape the development of Index-Coop. If you are interested in developing on `index-coop` or have any questions, please ping us on [Telegram](https://t.me/joinchat/Fx8D6wyprLUlM1jMVnaRdg).
```
Comment (cgewecke) [inline README.md:58]:
```suggestion
We highly encourage participation from the community to help shape the development of Index-Coop. If you are interested in developing on `index-coop` or have any questions, please ping us on [Discord](https://discord.com/invite/RKZ4S3b).
```

**Review by bweick — APPROVED** [review]

**Pull request merged**